vagrant/Vagrantfile

```diff
@@ -0,0 +1,69 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+box = { 
```

Comment:
Can you introduce the vagrant file in a different commit? The vagrant file is decoupled from the setup script that you're introducing above.

Also, please remove the commented lines.

Reply:
Fixed on 71271e15555acd05b640b863798813f77e3fdabc